advanced/lighttpd.conf.debian

```diff
@@ -52,10 +52,11 @@ $HTTP["url"] =~ "^/admin/" {
     )
 }
 
+# Rewite js requests, must be out of $HTTP block due to bug #2526
```

Comment:
I did not have any problem with javascripts getting rewritten with my alpine docker.

are you sure this isn't fixed in the newest alpine lighttpd?